libclangext/src/main/cpp/signalChaining.cpp

```diff
@@ -0,0 +1,58 @@
+#if defined(__linux__) || defined(__APPLE__)
```

Comment:
Can we avoid adding the same file twice?

Reply:
Btw, how does this help?
As far as I see, both copies should get linked into the same `libclangstubs` binary.

Reply:
Yeah, if we'll decide where this code goes. Suggestions?

Reply:
`Interop/Indexer`.
Having one copy is worth keeping it in somewhat unintuitive location.

Reply:
Ah, OK, maybe I was overly cautious here.

Reply:
Btw, we still may have to patch both `libclang` and `libclangstubs` (containing `clangext`).